transport/zipkin/http.go

```diff
@@ -160,7 +163,20 @@ func (c *HTTPTransport) send(spans []*zipkincore.Span) error {
 		req.SetBasicAuth(c.httpCredentials.username, c.httpCredentials.password)
 	}
 
-	_, err = c.client.Do(req)
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return err
+	}
+
+	defer resp.Body.Close()
+	_, err = io.Copy(ioutil.Discard, resp.Body)
+	if err != nil {
+		return err
+	}
```

Comment:
we are not using the response body here, so no need to drain it, only `callresp.Body.Close()` (https://forum.golangbridge.org/t/do-i-need-to-read-the-body-before-close-it/5594/3)

having said that, since the objective of this PR is to log the error that the server returned, logging the actual response (i.e. some error message), not just the status code, would make more sense.